src/_scss/pages/state/topFive/_table.scss

```diff
@@ -1,145 +1,177 @@
-            text-overflow: ellipsis;
-            overflow: hidden;
-            &:first-child {
-                text-align: left;
-                width: 55%
-            }
-            &:nth-child(2) {
-                width: 25%;
-            }
-            &:last-child {
-                width: 20%;
-            }
+      .tooltip-spacer {
+        @media(max-width: $tablet-screen) {
+          position:absolute;
+          left: 0 !important;
+          transform: translateX(-224px);
         }
+      }
+    }
+  }
 
-        .category-table__table-row{
-          .capitalize {
-            text-transform: capitalize;
-          }
-          &:last-child {
-            .category-table__table-cell {
-              border-bottom: 0;
-            }
-          }
-        }
+  .category-table__table {
+    font-size: $small-font-size;
+    border-collapse: separate;
+    border: solid $color-gray-border rem(1);
+    border-radius: rem(5);
+    -moz-border-radius: rem(5);
+    margin: 0;
+
+    .category-table__table-head-cell {
+      padding: rem(10) rem(15);
+      border: 0;
+      background-color: transparent;
+      border-bottom: solid $color-gray-border rem(1);
+      font-weight: 600;
+      text-align: center;
 
+      &:first-child {
+        text-align: left;
+      }
     }
 
-    .category-table__message {
-        color:$color-gray-light;
-        font-size: rem(18);
-        text-align: center;
-        background-color: #FCFCFC;
-        height: 100%;
-        position: relative;
-        border-radius: rem(5);
-        min-height: 68px;
-        border: rem(1) solid #D5D6D8;
-        span {
-        	position: absolute;
-        	top: 35%;
-        	left: 40%;
+
+    .category-table__table-cell {
+      padding: rem(10) rem(15);
+      border: 0;
+      border-bottom: solid rgba(174, 176, 181, 0.5) rem(1);
+      background-color: transparent;
+      font-weight: 400;
+      text-align: center;
+      max-width: 0;
+      white-space: nowrap;
+      text-overflow: ellipsis;
+      overflow: hidden;
+
+      &:first-child {
+        text-align: left;
+        width: 55%
+      }
+
+      &:nth-child(2) {
+        width: 25%;
+      }
+
+      &:last-child {
+        width: 20%;
+      }
+    }
+
+    .category-table__table-row {
+      .capitalize {
+        text-transform: capitalize;
+      }
+
+      &:last-child {
+        .category-table__table-cell {
+          border-bottom: 0;
         }
+      }
     }
 
-    .category-table__disclaimer {
-      font-size: 75%;
+  }
+
+  .category-table__message {
+    color: $color-gray-light;
+    font-size: rem(18);
+    text-align: center;
+    background-color: #FCFCFC;
+    height: 100%;
+    position: relative;
+    border-radius: rem(5);
+    min-height: 68px;
+    border: rem(1) solid #D5D6D8;
+
+    span {
+      position: absolute;
+      top: 35%;
+      left: 40%;
     }
+  }
+
+  .category-table__disclaimer {
+    font-size: 75%;
+  }
 
   .category-table__link {
```

Comment:
It looks like this class is no longer used anywhere and can be removed

Reply:
removed